README.rst

```diff
@@ -30,7 +30,7 @@ In order to use this library, you first need to go through the following steps:
 
 .. _Select or create a Cloud Platform project.: https://console.cloud.google.com/project
 .. _Enable billing for your project.: https://cloud.google.com/billing/docs/how-to/modify-project#enable_billing_for_a_project
-.. _Enable the Cloud BigQuery Reservation API.:  https:///cloud.google.com/bigquery/docs/reference/reservations
+.. _Enable the Cloud BigQuery Reservation API.:  https:///cloud.google.com/bigquery/docs/reference/reservations/rpc
```

Comment:
Can you grab this extra slash, please?

Reply:
Not sure I understand.  You want the RPC URL slash terminated?

Reply:
Sorry I wasn't more clear. There's an extra slash in the beginning of the URL. https:///...